inugami_webapp/src/main/webapp/css/core/component-gitlab.css

```diff
@@ -0,0 +1,105 @@
+
+.gitlab-merge-requests>ul{
+    list-style: none;
+    margin: 0;
+    padding: 0;
+}
+
+.gitlab-merge-requests article{
+    clear: both;
+    margin-bottom: 0;
+    font-size: 90%;
+    padding-bottom: 0.2em;
+    border-bottom: 1px solid #e0d7d7;
+    padding-left: 0.5em;
+}
+
+
+.gitlab-merge-requests article header .project{
+    font-weight: bold;
+}
+
+.gitlab-merge-requests article  .title{
+    font-size: 80%;
+}
+
+.gitlab-merge-requests article .more-informations{
+    font-size: 70%;
+    color: #888888;
+}
+
+.gitlab-merge-requests article .created,
+.gitlab-merge-requests article .author{
+    float: left;
+}
+.gitlab-merge-requests article .created{
+
+}
+
+.gitlab-merge-requests article .author::before{
+    content:" : ";
+}
+
+
+.gitlab-merge-requests article .votes{
+    float: right;
+}
+
+.gitlab-merge-requests article .votes .vote{
+    border: 1px solid gray;
+    display: block;
+    text-align: right;
+    border-radius: 1em;
+    float: right;
+    margin-right: 0.5em;
+}
+
+.gitlab-merge-requests article .votes .vote-icon{
+    -webkit-font-smoothing: antialiased;
+    display: inline-block;
+    font-style: normal;
+    font-variant: normal;
+    text-rendering: auto;
+    line-height: 1;
+    font-family: Font Awesome\ 5 Free;

```

Comment:
Ajouter un fond par défaut si il manque ? Serif ?

Reply:
Why you recommends a default backgrounds ?  For serif font, we use default font define in application.css.